scripts/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
Does this need to change to bash? 

This was previously changed to bash but had to be reverted as the official docker image runs alpine and does not have bash installed.

Reply:
Then you shouldn't use [[ ]], sh doesn't know this syntax. I reverted the shebang in the following commit and replaced the [[ ]] by simple [ ], which works too

Reply:
Thanks! My sh/bash skills aren't great but the scripts have seemed to work in my environments so far. Thanks for fixing this too :)